etc/profile-m-z/telegram.profile

```diff
@@ -7,6 +7,7 @@ include globals.local
 
 noblacklist ${HOME}/.TelegramDesktop
 noblacklist ${HOME}/.local/share/TelegramDesktop
+noblacklist ${HOME}/.local/share/telegram-desktop
```

Comment:
This needs a `blacklist ${HOME}/.local/share/telegram-desktop` counterpart in `disable-programs.inc` to protect the path from leaking into other sandboxes. Respecting alphabetical sorting it should go right after `blacklist ${HOME}/.local/share/swell-foop`.